### PR TITLE
DTGB-402: Added a required fields indicator to forms.

### PR DESCRIPTION
**Pull request by meirege**
## Description
As described on https://digipolisgent.atlassian.net/wiki/spaces/HSSG/pages/11960359/Formulier+elementen
Form fields require an indicator to tell the user all fields are required unless stated otherwise.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I have updated the documentation accordingly.
- [x] I have updated the style guide CHANGELOG accordingly.
- [x] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: styleguide/components/51-templates/forms/forms--with-errors.twig [hunk 1]
```diff
@@ -1,6 +1,10 @@
 <h2 class="styleguide-title">Forms</h2>
 <div class="preview-inner-container">
   <form class="{{ modifier }}">
+    {% include '@paragraph' with {
+      'text' : 'Vul alle velden in tenzij anders vermeld',
```
Comment (Jeroen005) [inline styleguide/components/51-templates/forms/forms--with-errors.twig]: English :)

File: styleguide/components/51-templates/forms/forms.twig [hunk 2]
```diff
@@ -1,6 +1,10 @@
 <h2 class="styleguide-title">Forms</h2>
 <div class="preview-inner-container">
   <form class="{{ modifier }}">
+    {% include '@paragraph' with {
+      'text' : 'Vul alle velden in tenzij anders vermeld',
```
Comment (Jeroen005) [inline styleguide/components/51-templates/forms/forms.twig]: English :)

File: styleguide/components/61-layouts/forms/form-with-steps/form-with-steps.twig [hunk 3]
```diff
@@ -59,75 +59,80 @@
...
-                'label': 'Describe your activity and it\'s purpose',
-                'description': 'Describe your activity and it\'s purpose.',
-                "field_error": 'Please shortly describe your activity.',
-                "field_description": 'A short and simplified description of your activity.'
-              }
-            %}
-
+        <form action="">
+          {% include '@paragraph' with {
+            'text' : 'Vul alle velden in tenzij anders vermeld',
```
Comment (Jeroen005) [inline styleguide/components/61-layouts/forms/form-with-steps/form-with-steps.twig]: English :)

File: styleguide/components/61-layouts/forms/form-with-steps/form-with-steps.twig [hunk 4]
```diff
@@ -59,75 +59,80 @@
...
-                "radio1_label_for": "radio-1",
-                "radio1_label_optional": "",
+                  "field_description": "",
+                  "field_error": "",
+                  "description": "",
 
-                "label_radio2": "Ik reserveer deze zaal als particulier of feitelijke vereniging",
-                "radio2_name": "radiogroup",
-                "radio2_id": "radio-2",
+                  "label_radio1": "Ik reserveer deze zaal als particulier of feitelijke vereniging",
```
Comment (Jeroen005) [inline styleguide/components/61-layouts/forms/form-with-steps/form-with-steps.twig]: English :)

File: styleguide/components/61-layouts/forms/form-with-steps/form-with-steps.twig [hunk 5]
```diff
@@ -59,75 +59,80 @@
...
+                  "radio1_name": "radiogroup",
+                  "radio1_id": "radio-1",
 
-                "radio2_label": "Ik reserveer deze zaal als vennootschap of een vereniging met rechtspersoonlijkheid",
-                "radio2_label_for": "radio-2",
-                "radio2_label_optional": ""
-              } %}
+                  "radio1_label": "Ik reserveer deze zaal als particulier of feitelijke vereniging",
```
Comment (Jeroen005) [inline styleguide/components/61-layouts/forms/form-with-steps/form-with-steps.twig]: English :)

File: styleguide/components/61-layouts/forms/form-with-steps/form-with-steps.twig [hunk 6]
```diff
@@ -59,75 +59,80 @@
...
+                  "radio1_label_for": "radio-1",
+                  "radio1_label_optional": "",
 
-              {% include '@address' %}
+                  "label_radio2": "Ik reserveer deze zaal als particulier of feitelijke vereniging",
```
Comment (Jeroen005) [inline styleguide/components/61-layouts/forms/form-with-steps/form-with-steps.twig]: English :)

File: styleguide/components/61-layouts/forms/form-with-steps/form-with-steps.twig [hunk 7]
```diff
@@ -59,75 +59,80 @@
...
+                  "radio2_name": "radiogroup",
+                  "radio2_id": "radio-2",
+
+                  "radio2_label": "Ik reserveer deze zaal als vennootschap of een vereniging met rechtspersoonlijkheid",
```
Comment (Jeroen005) [inline styleguide/components/61-layouts/forms/form-with-steps/form-with-steps.twig]: English :)

**Review by Jeroen005 — CHANGES_REQUESTED** [review]

File: styleguide/components/51-templates/forms/forms--with-errors.twig (hunk 1 above)
Comment (meirege) [reply styleguide/components/51-templates/forms/forms--with-errors.twig]: Fixed.

File: styleguide/components/51-templates/forms/forms.twig (hunk 2 above)
Comment (meirege) [reply styleguide/components/51-templates/forms/forms.twig]: Fixed.

File: styleguide/components/61-layouts/forms/form-with-steps/form-with-steps.twig (hunk 3 above)
Comment (meirege) [reply styleguide/components/61-layouts/forms/form-with-steps/form-with-steps.twig]: Fixed.

File: styleguide/components/61-layouts/forms/form-with-steps/form-with-steps.twig (hunk 4 above)
Comment (meirege) [reply styleguide/components/61-layouts/forms/form-with-steps/form-with-steps.twig]: Fixed.

File: styleguide/components/61-layouts/forms/form-with-steps/form-with-steps.twig (hunk 5 above)
Comment (meirege) [reply styleguide/components/61-layouts/forms/form-with-steps/form-with-steps.twig]: Fixed.

File: styleguide/components/61-layouts/forms/form-with-steps/form-with-steps.twig (hunk 6 above)
Comment (meirege) [reply styleguide/components/61-layouts/forms/form-with-steps/form-with-steps.twig]: Fixed.

File: styleguide/components/61-layouts/forms/form-with-steps/form-with-steps.twig (hunk 7 above)
Comment (meirege) [reply styleguide/components/61-layouts/forms/form-with-steps/form-with-steps.twig]: Fixed.

**Review by Jeroen005 — APPROVED** [review]

**Pull request merged**